prajwal-vw-header/includes/css/scss/_module-header-logo-dealer-name.scss

```
@@ -0,0 +1,59 @@
+.brandLogoDealerNameWrapper {
+	display: grid;
+	grid-template-columns: 1fr minmax(auto, $container-width) 1fr;
+	align-items: center;
+
+	&::before,
+	&::after {
+		content: "";
+		border-top: 2px solid $white;
```

Use a variable to control the line width so it always changes together.
```scss
$brandLineWidth: 2px;
```

```suggestion
		border-top: $brandLineWidth solid $white;
```

---

Resolved based on commit e0939d3